### PR TITLE
upload fastp gencore and mutscan

### PR DESCRIPTION
**Pull request by zhouyangwang**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

 * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
 * `@BiocondaBot please add label` will add the `please review & merge` label.
 * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.

For members of the Bioconda project, the following command is also available:

 * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.

If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by dpryan79** [comment]
Please disable circleci on your fork.

**Comment by zhouyangwang** [comment]
> Please disable circleci on your fork.



> Please disable circleci on your fork.

I have already disable circleci on my fork. Please review

**Comment by dpryan79** [comment]
@bioconda-bot please update

**Comment by dpryan79** [comment]
Crap, I've run into the current error before and have to remember how to fix it. Please note that the OSX failure has nothing to do with your code, it's some weird combination of clang and the CI infrastructure on circleci.

@mbargull I recall errors like this having to do with sysroot. Do you have any ideas?

**Comment by mbargull** [comment]
Hm.. maybe the SDK from the newer image is that different that we do have copy the conda-forge CI thing? I'll try that.

File: .circleci/setup.sh [hunk 1]
```diff
@@ -70,6 +70,15 @@ if ! type bioconda-utils 2> /dev/null || [[ $BOOTSTRAP == "true" ]]; then
     $WORKSPACE/miniconda/bin/conda install -y git pip --file https://raw.githubusercontent.com/bioconda/bioconda-utils/$BIOCONDA_UTILS_TAG/bioconda_utils/bioconda_utils-requirements.txt
     $WORKSPACE/miniconda/bin/pip install git+https://github.com/bioconda/bioconda-utils.git@$BIOCONDA_UTILS_TAG
 
+    if [[ $OSTYPE == darwin* ]]; then
```
Comment (mbargull) [inline .circleci/setup.sh]: @dpryan79, I cherry-picked this into https://github.com/bioconda/bioconda-recipes/pull/19610.

**Comment by dpryan79** [comment]
@bioconda-bot please update

File: .circleci/setup.sh (hunk 1 above)
Comment (dpryan79) [reply .circleci/setup.sh]: Thanks, hopefully that will solve things :)

**Comment by mbargull** [comment]
> @bioconda-bot please update

Don't you call me bot! ;P

**Comment by dpryan79** [comment]
LOL

**Comment by dpryan79** [comment]
I don't understand why the last commit failed on OSX, the only thing that changed were comments in the CI stuff!?!

**Comment by mbargull** [comment]
Maybe there was a system-provided Conda environment active and the changes from `run_conda_forge_build_setup` were not written to the cached Conda installation?
Let's merge gh-19612 when it's green and try again.

**Comment by dpryan79** [comment]
Worth a go.

**Comment by mbargull** [comment]
Yep, it showed `base environment : /Users/distiller/project/miniconda`.
At some point we should clean up some stuff and properly activate our environment in `setup.sh`.
But it should work now.

**Comment by mbargull** [comment]
@bioconda-bot please update

**Comment by dpryan79** [comment]
Boo :(

**Comment by mbargull** [comment]
FFS :/. Gonna give it another try in a few minutes.

**Comment by mbargull** [comment]
gh-19614 is my next attempt...

**Comment by dpryan79** [comment]
You might try directly modifying this PR instead, since that will allow testing the effect of the changes.

**Comment by mbargull** [comment]
I didn't modify this one to yet so we can see whether it runs fine when running from the cache.

**Comment by mbargull** [comment]
But instead of merging the other one, I can push those changes there afterwards (but before changing `master`) too, because the cache will be up then, I guess.

**Comment by dpryan79** [comment]
I'd forgotten about the cache...

**Comment by mbargull** [comment]
And I always tend to forget that the cache is not solely populated from `master` ;).

**Comment by dpryan79** [comment]
Oh, I hadn't even realized that...

**Comment by dpryan79** [comment]
Hmm, I wish conda would at least produce an error message...

**Comment by mbargull** [comment]
> Maybe there was a system-provided Conda environment active \[...\]

> Yep, it showed base environment : `/Users/distiller/project/miniconda` \[...\]

Hm, no I was confused, that was ours, but had `conda 4.8.0` since that had been installed alongside `conda-forge-ci-setup`.

Debugging...

**Comment by mbargull** [comment]
Oh, goddammit, I'm a dummy. The SDK is outside of the cache, of course. Meaning we have to always run `run_conda_forge_build_setup`.
Okay, I'll make that last one fix and then it should be fine, I hope!

**Comment by mbargull** [comment]
@bioconda-bot please update

**Comment by mbargull** [comment]
Well, this was a little more bump than anticipated/needed :sweat_smile:.
@dpryan79, feel free to continue with business as usual!

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**